app/src/main/res/layout/weather_card_item.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/card_item"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:cardCornerRadius="@dimen/border_radius_card_big"
+    app:cardElevation="@dimen/card_elevation_xsmall"
+    app:cardUseCompatPadding="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/constraint"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_gravity="center">
+
+        <ImageView
+            android:id="@+id/weather_image_view"
+            android:layout_width="@dimen/width_image_view_xbig"
+            android:layout_height="@dimen/height_image_view_xbig"
+            android:layout_marginLeft="@dimen/margin_left_image_view_normal"
+            app:layout_constraintBottom_toBottomOf="@id/constraint"
+            app:layout_constraintStart_toStartOf="@id/constraint"
```

Comment:
In here, instead of using the constraints id as a reference, use "parent", to make it less confusing.
Apply it on the rest of the constraints
```suggestion
            app:layout_constraintStart_toStartOf="parent"
```

Reply:
References of constraints layout made.